tasks/Functions-admin/deployFullOracle.js

```diff
@@ -70,62 +77,48 @@ task(
   console.log("Registry configuration set")
 
   // Set up Functions Oracle
```

Comment:
Do we only need to call `acceptOwnership` when the proxy contract is deployed and not when the contract is upgraded?

Also, I assume that we are deploying the proxy contracts manually first, then entering the proxy addresses into `network-config.js` before running `functions-deploy-oracle`?

Reply:
I am still not completely sure I understand the deployment logic, but I will spend more time reviewing & ask relevant questions once this PR moves from `draft` to `open`.

It might be worth a 15 minute sync to explain too.  Whatever is easier.

Reply:
Let's sync because deployment timing is a little weird.

Reply:
> Do we only need to call acceptOwnership when the proxy contract is deployed and not when the contract is upgraded?

Yes. The hardhat upgrades plugin handles all of the ownership for us.

> Also, I assume that we are deploying the proxy contracts manually first, then entering the proxy addresses into network-config.js before running functions-deploy-oracle?

No, `functions-deploy-oracle` deploys the proxy admin, proxy, and imlementation - then it spits back out the addresses for us to updates `network-config.js`.